client/landing/jetpack-cloud/components/log-item/index.tsx

```diff
@@ -0,0 +1,52 @@
+/**
+ * External dependencies
+ */
+import React, { ReactNode } from 'react';
+
+/**
+ * Internal dependencies
+ */
+import FoldableCard from 'components/foldable-card';
+import CardHeading from 'components/card-heading';
+
+import './style.scss';
+
+interface Props {
+	children?: ReactNode;
+	header: string;
+	subheader: string;
+	highlight?: string;
```

Comment:
Should we limit this to the expected values? That's what we do in `Card` for example:

```
highlight: PropTypes.oneOf( [ 'error', 'info', 'success', 'warning' ] ),
```

Reply:
Resolved in 778e04f65beafd3882fa8a6eac29d6b491c0bf03. I used Typescript instead of `PropTypes`.